### PR TITLE
Add webhook notification page to references and guided tour

### PR DESCRIPTION
**Pull request by radwo**
[doc is already on production ](https://docs.semaphoreci.com/guided-tour/webhook-notifications/) in this PR I adding it to `guided-tour` and `reference` sections, just like `slack-notifications` note.

### CODE REVIEW

**Comment by shiroyasha** [comment]
@Ivana-Markovic I agree with most. Can you update your comments with Github inline comments? (find the line that is needs to be modified > click + > add your comment)?

It is very hard to follow the comments without that approach.

**Comment by Ivana-Markovic** [comment]
Will do @shiroyasha (thanks for teaching me, I did not know how to do that (blush)).

File: docs/reference/slack-notifications.md [hunk 1]
```diff
@@ -212,8 +212,8 @@ for further details.
 
 ## Setting up, editing and deleting Slack notifications through the UI
 
-In the Configuration part of the sidebar, click on **Notifications** -> **Create New 
-Notification**. Add the name of the notification and rules and click on the **Save 
+In the Configuration part of the sidebar, click on **Notifications** -> **Create New
+Notification**. Add the name of the notification and rules and click on the **Save
```
Comment (Ivana-Markovic) [inline docs/reference/slack-notifications.md]: Add the name of the notification and rules, then click on the

File: docs/reference/webhook-notifications.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
+# Webhook Notifications
+
+Semaphore has webhook based notifications that are delivered on the success or
+failure of a pipeline. When criteria for notification are met, Semaphore will
```
Comment (Ivana-Markovic) [inline docs/reference/webhook-notifications.md]: When criteria for notifications are met

File: docs/reference/webhook-notifications.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+``` bash
+$ sem create notification master-pipelines \
+    --projects web \
+    --branches master \
+    --webhook-endpoint [webhook-endpoint]
+```
+
+## Setting up webhook notifications for multiple projects
+
+When creating a notification, you can specify multiple projects as source,
```
Comment (Ivana-Markovic) [inline docs/reference/webhook-notifications.md]: When creating a notification, you can specify multiple projects as a source

File: docs/reference/webhook-notifications.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+of your notifications.
+
+For example, if your team manages three projects named `web`, `cli` and `api`
+and you want to get notified for every finished pipeline on the master branch,
```
Comment (Ivana-Markovic) [inline docs/reference/webhook-notifications.md]: and you want to get notified of every finished pipeline on the master branch,

File: docs/reference/webhook-notifications.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+    --branches "master" \
+    --webhook-endpoint [webhook-endpoint]
+```
+
+## Filtering by project, branch and pipeline names
+
+When creating a notification, you can specify a filter for project, branch and
+pipeline names.
+
+For example to send notifications for the `master` and `staging` branches use
```
Comment (Ivana-Markovic) [inline docs/reference/webhook-notifications.md]: For example, to send notifications for the `master` and `staging` branches, use

File: docs/reference/webhook-notifications.md [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+forward slashes (e.g. `master`) defines a direct match.
+
+``` bash
+$ sem create notifications example \
+    --branches "master,/hotfix\/.*/" \
+    --webhook-endpoint [webhook-endpoint] \
+```
+
+Matching can be specified for project and pipeline names as well. For example,
+if you want to get notified about every notification on a project that matches
```
Comment (Ivana-Markovic) [inline docs/reference/webhook-notifications.md]: Please reword it to avoid 'notified about every notification' - I can help but we'd need to sync first.

File: docs/reference/webhook-notifications.md [hunk 7]
```diff
@@ -0,0 +1,274 @@
...
+                    "id": "d4b93a5b-69a5-43e6-ab24-06b095fc49bf"
+                }
+            ]
+        }
+    ]
+}
+```
+
+In this example `revision.pull_request` and `revision.tag` are null because
+payload is related to pipeline run started from a push to the branch. Information
```
Comment (Ivana-Markovic) [inline docs/reference/webhook-notifications.md]: payload is related to the pipeline run started from a push to the branch.

**Comment by radwo** [comment]
@Ivana-Markovic thanks for this feedback :)

File: docs/guided-tour/webhook-notifications.md [hunk 8]
```diff
@@ -30,11 +30,11 @@ $ sem create notification master-pipelines \
 
 ## Setting up webhook notifications for multiple projects
 
-When creating a notification, you can specify multiple projects as source,
+When creating a notification, you can specify multiple projects as a source,
```
Comment (Ivana-Markovic) [inline docs/guided-tour/webhook-notifications.md]: "a source," please remove the comma (,) after the source
